docs/pages/careers/design-engineer-x-grid.md

```diff
@@ -88,10 +88,11 @@ The Design Engineer role sits right at the intersection between product design a
 ### Required
 
 - 3+ years of web front-end development at product-driven companies
+- Advanced knowledge of modern React, HTML, and CSS
```

Comment:
I have seen candidates who have no engineering background. This doesn't make sense relative to the role. For example https://app.ashbyhq.com/jobs/c66c2524-bcdb-4369-b1b4-be3cc4ef21be/candidate-pipeline/application-review/right-side/candidates/d8cec097-b2f1-45e2-a95c-e571620d2836/applications/f66eb4a9-1cef-4c9a-98ad-53fd2588b920/feed/